app/views/course/assessment/submission/submissions/_guided.html.slim

```diff
@@ -1,16 +1,32 @@
-nav
-  ul.pagination
-    - (1..@assessment.questions.length).each do |i|
-      li class="#{guided_nav_class(i)}"
-        = link_to i, [:edit, current_course, @assessment, @submission, step: i], class: ['btn', 'btn-default']
+- if @submission.attempting?
```

Comment:
then staff will do grading step by step or ? that does not make sense to me actually..